src/copyrightinserter.ts

```diff
@@ -191,15 +191,42 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.`]
         return line;
     }
 
+    private formatString(header: string, first_line: string, prefix: string, last_line: string): string {
+        var cur_prefix = first_line
```

Comment:
consider adding trailing semicolon (;) to the new lines. VSCode shows warning for these lines because the file type is typescript and not javascript.

Reply:
Done.